week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,59 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
 
 // The answer should come from shaking the ball
-let answer;
+
 
 // When checking the answer, we should tell someone if the answer is
 // - very positive
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
+function checkAnswer(message) {
```

Comment:
This `checkAnswer` function is working well, good job!

Unfortunately when I run the tests they're not quite passing yet:
```
% node week-1/Homework/mandatory/5-magic-8-ball.js                                                                                                                                             √
/Users/dwh/src/github.com/CodeYourFuture/js-exercises/week-1/Homework/mandatory/5-magic-8-ball.js:124
  const answer = shakeBall();
                 ^

ReferenceError: shakeBall is not defined
    at testAll (/Users/dwh/src/github.com/CodeYourFuture/js-exercises/week-1/Homework/mandatory/5-magic-8-ball.js:124:18)
    at Object.<anonymous> (/Users/dwh/src/github.com/CodeYourFuture/js-exercises/week-1/Homework/mandatory/5-magic-8-ball.js:138:1)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
```

Can you try out finishing them off? Happy to chat if you need a hand!